config/taxonomy/policymanager_response.structs.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "opa.schema.json",
+    "title": "OPA Data Response Taxonomy",
+    "description": "This taxonomy defines the structure of response received from Policy Agent policy engine with enforcements.",
```

Comment:
change description because this isn't OPA specific

Reply:
Changed the description